app/views/booths/index.html.haml

```diff
@@ -0,0 +1,51 @@
+.container
+  .row
+    .col-md-12.page-header
+      %h1
+        Your booth requests for
+        = @conference.title
+  .row
+    .col-md-12
+      .margin-booth-table
+        %table.table.table-striped.table-hover
+          %thead
+            %th
+              %b State
+            %th
+              %b Logo
+            %th
+              %b Title
+            %th
+              %b Actions
+          - @booths.each do |booth|
+            %tr
+              %td{style: "padding:20px 8px 20px 8px;"}
+                - if (booth.state == 'to_accept' || booth.state == 'to_reject')
+                  - show_state = 'new'
+                - else
+                  - show_state = booth.state
+                %span{ title: show_state, class: "fa #{booth_status_icon(booth)}" }
+              %td
+                - if booth.logo_link
+                  = image_tag(booth.picture.thumb.url, width: '20%')
+              %td
+                = link_to booth.title, conference_booth_path(@conference.short_title, booth)
+              %td
+                -if can? :edit, booth
+                  = link_to 'Edit', edit_conference_booth_path(@conference.short_title, booth.id),
+                    class: 'btn btn-primary'
```

Comment:
There was also a second comment about the color scheme. Let's follow the colors used in proposals#index